src/parsers/manifestjson.js

```diff
@@ -37,28 +37,48 @@ export default class ManifestJSONParser {
     this.isValid = this._validate();
   }
 
+  errorLookup(error) {
+    var result = {
```

Comment:
This is odd as it appears to be a message that isn't in the `message.js` file. We kept everything there so we'd be able to trace where all messages came from. I'd start with a message there and add your dynamically built `message` attribute to it here.